### PR TITLE
More fixes for cordova. integrating real contacts

### PR DESCRIPTION
**Pull request by k88hudson**

### CODE REVIEW

File: components/contactPicker/index.js [hunk 1]
```diff
@@ -4,17 +4,23 @@ module.exports = {
...
+        var self = this;
+        self.modelData();
 
-        this.$on('openContactPicker', function (event) {
-            this.open();
+        self.$on('openContactPicker', function (event) {
+            self.$data.contacts = []
+            navigator.contacts.find(['*'], function (contacts) {
+                console.log(contacts);
+                contacts.forEach(function (contact) {
```
Comment (k88hudson) [inline components/contactPicker/index.js]: I can't get vue.js to frikin update the UI in this async function. kill me now

Comment (k88hudson) [inline components/contactPicker/index.js]: any ideas @gvn my brain is dead

File: static/styles/lists.less [hunk 2]
```diff
@@ -55,7 +55,7 @@
 
     // For left and right sections
     .left, .right {
-      float: left;
+     // float: left;
```
Comment (gvn) [inline static/styles/lists.less]: Just delete this?

**Comment by k88hudson** [comment]
@thisandagain or @gvn R?

File: components/contactPicker/index.js [hunk 3]
```diff
@@ -37,12 +39,17 @@ module.exports = {
             var modeled = {};
 
             // Sort contacts into alphabetical order
-            this.contacts = this.contacts.sort(function (a, b) {
+            this.contacts = this.contacts
+            .filter(function (contact) {
```
Comment (thisandagain) [inline components/contactPicker/index.js]: :+1:

File: static/index.html [hunk 4]
```diff
@@ -2,7 +2,11 @@
 <html<% if (APPCACHE) { %> manifest="manifest.appcache"<% } %>>
 
     <head>
-        <base href="<%= BASE_URL %>">
+        <script>
```
Comment (thisandagain) [inline static/index.html]: :+1:

File: views/share/index.js [hunk 5]
```diff
@@ -11,7 +11,11 @@ module.exports = view.extend({
     data: {
         title: 'Share',
         error: false,
-        doneDisabled: true
+        doneDisabled: true,
+        showAndroid: false,
```
Comment (thisandagain) [inline views/share/index.js]: Might want to use more generic language here... `showDevice`?

**Comment by thisandagain** [comment]
Looks good. One small note. R+

File: views/share/index.js (hunk 5 above)
Comment (k88hudson) [inline views/share/index.js]: actually this should be `showPicker`, sorry, my bad

**Pull request merged**